src/Api/axiosInstance.tsx

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+const axiosInstance = axios.create({
+  baseURL: 'http://localhost:8000',
+  withCredentials: true,
+  timeout: 5000,
+  headers: {
+    'Content-Type': 'application/json',
+  },
+});
+
+export default axiosInstance;
```

Comment:
name it better like baseApi (file and axios object)
then just use it for the others endpoints